blogPosts/psrules-june-2022.md

```diff
@@ -0,0 +1,144 @@
+
+### PsRules For Azure - (Intermediate)
+
+So we now know that PsRules for Azure can verify our IAC templates against best practices. But what is it scanning for? Well, it includes over 260 rules that it evaluates for. For a list of what is evaluated [click here](https://azure.github.io/PSRule.Rules.Azure/en/rules/module/). 
+
+Finally, let's dive into how to use this tool!
+
+#### Requirements
+
+The following platforms are supported:
+
+- Windows PowerShell 5.1 with .NET Framework 4.7.2 or greater.
+- PowerShell 7.2 or greater on MacOS, Linux, and Windows.
+
+#### Step 1 - Installation
+
+PsRules for Azure can be installed locally by running the following commands in a PowerShell terminal:
+
+To install PSRule for Azure for the current user use:
+
+    ```powershell
+    Install-Module -Name 'PsRule' -Repository PSGallery -Scope CurrentUser -Force
+    Install-Module -Name 'PSRule.Rules.Azure' -Repository PSGallery -Scope CurrentUser -Force
+    ```
+
+To enable these new modules for the current user use:
+
+    ```powershell
+    Update-Module -Name 'PSRule'
+    Update-Module -Name 'PSRule.Rules.Azure'
+    ```
+
+> *Note:* For the full installation instructions (local, Azure Devops & GitHub) use this [link](https://azure.github.io/PSRule.Rules.Azure/install-instructions/#installation)
+
+#### Step 2 - Demo Setup
+
+Now that the tool is installed we can start preparing to scan our IAC templates. Below is the code for the template I will be scanning in this demo ([link](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/quickstarts/microsoft.storage/storage-account-create/main.bicep)):
+
+    ```bicep
+    // Deploy storage account
+    @description('Storage Account type')
+    @allowed([
+        'Premium_LRS'
+        'Premium_ZRS'
+        'Standard_GRS'
+        'Standard_GZRS'
+        'Standard_LRS'
+        'Standard_RAGRS'
+        'Standard_RAGZRS'
+        'Standard_ZRS'
+    ])
+    param storageAccountType string = 'Standard_LRS'
+
+    @description('Location for the storage account.')
+    param location string = resourceGroup().location
+
+    @description('The name of the Storage Account')
+    param storageAccountName string = 'store${uniqueString(resourceGroup().id)}'
+
+    resource sa 'Microsoft.Storage/storageAccounts@2021-06-01' = {
+    name: storageAccountName
+    location: location
+    sku: {
+        name: storageAccountType
+    }
+    kind: 'StorageV2'
+    properties: {}
+    }
+    ```
+
+This file is available locally on my device in a folder called `examples`. Before we can test the Bicep module we need to test create a file called `ps-rule.yaml` with the following content:
+
+    ```yaml
+    # YAML: Enable expansion for Bicep source files.
+    configuration:
+    AZURE_BICEP_FILE_EXPANSION: true
+    ```
```

Comment:
```suggestion
    ```yaml
    # YAML: Enable expansion for Bicep source files.
    configuration:
      AZURE_BICEP_FILE_EXPANSION: true
    ```
```